packages/demand-bloc-track-trip/src/TripBloc.ts

```diff
@@ -25,13 +30,25 @@ export interface Storage {
   getItem: (key: string) => string | null
 }
 
-export type TripService = Pick<Trip, 'cancelByFollowCode' | 'trackTrip'>
+export type TripService = Pick<Trip, 'cancelByFollowCode' | 'trackTrip' | 'search' | 'cancel'>
 export type FareService = Pick<Fare, 'status'>
+export type TripsOffset = {
+  upcomingTripsOffset: number
+  pastTripsOffset: number
+}
+export type CustomOptions = {
+  paginationRowCount: {
+    upcomingTrips: number
+    pastTrips: number
+  }
+}
```

Comment:
Is nesting really necessary here? I'd avoid nesting unless there are strong reasons for it.